colloids+squirmers+subgrid particles

qikaifzj opened the pull request:
This repository can accommodate colloids, squirmers, and subgrid particles simultaneously. It can also be used to study each element individually.

kevinstratford commented:
Thanks, I'll take a look at this.

Couple of points.
1. Could you make the pull request to the "develop" branch. We don't push changes
straight to the master.
2. Comments are /* C style */ and probably try to avoid such comments as "CHANGE"
as not very useful.

qikaifzj commented:
Dear Kevin,

Thank you very much for your response!

I am sorry that I forgot to delete ‘CHANGE’ in the first place. It was used by me initially when I modified the code. I have deleted all of them and sent a new pull request to the “develop”.

Thank you very much for your help! I am looking forward to your feedback!

Best,
Kai

> On Apr 8, 2020, at 5:56 PM, kevinstratford <notifications@github.com> wrote:
> 
> 
> Thanks, I'll take a look at this.
> 
> Couple of points.
> 
> Could you make the pull request to the "develop" branch. We don't push changes
> straight to the master.
> Comments are /* C style */ and probably try to avoid such comments as "CHANGE"
> as not very useful.
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/ludwig-cf/ludwig/pull/75#issuecomment-611041646>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AKMG3QSMYNQ6UMOVIKOJ4EDRLSNDNANCNFSM4MDNZUCA>.
>

kevinstratford commented:
This is a repeat, so I will close it and use #76
